lms/djangoapps/onboarding_survey/forms.py

```diff
@@ -25,6 +27,12 @@
 no_option_select_error = 'Please select an option for {}'
 empty_field_error = 'Please enter your {}'
 
+def get_data_from_json_file(file_name):
```

Comment:
Please update method name for more appropriate meanings. like get_onboarding_autosuggesion_data